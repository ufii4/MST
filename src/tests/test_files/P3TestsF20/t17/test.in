n 3
i 0;0;1
i 1;1;1
i 2;2;1
degree 0
degree 1
degree 2
edge_count
mst
clear
edge_count
mst
i 0;0;100.00
i 1;1;100.00
i 2;2;100.00
i 0;1;50.5
degree 0
degree 1
degree 2
edge_count
mst
i 1;2;100.05
degree 0
degree 1
degree 2
edge_count
mst
d 1;2
degree 0
degree 1
degree 2
edge_count
mst
i 0;2;100.05
degree 0
degree 1
degree 2
edge_count
mst
clear
edge_count
mst
exit
