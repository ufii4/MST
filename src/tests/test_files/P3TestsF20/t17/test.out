success
success
success
success
degree of 0 is 2
degree of 1 is 2
degree of 2 is 2
edge count is 3
not connected
success
edge count is 0
not connected
success
success
success
success
degree of 0 is 3
degree of 1 is 3
degree of 2 is 2
edge count is 4
not connected
success
degree of 0 is 3
degree of 1 is 4
degree of 2 is 3
edge count is 5
mst 150.55
success
degree of 0 is 3
degree of 1 is 3
degree of 2 is 2
edge count is 4
not connected
success
degree of 0 is 4
degree of 1 is 3
degree of 2 is 3
edge count is 5
mst 150.55
success
edge count is 0
not connected
